### ISSUE TITLE
How to have none inputs for filepath upload and mic

### ISSUE THREAD

**Issue by sciai-ai** [gradio-app/gradio#1613]
Running the following code snippet with examples gives me a filepath error as the filepath doesnot exist. It appears type can only be `numpy` or `filepath`. How can i resolve this?

``` python
inputs= [gr.inputs.Audio(type="filepath", source="upload", optional=True),
    gr.inputs.Audio(type="filepath", source="microphone", optional=True)],`

examples=[[i,None] for i in glob("./examples/*.wav")],

```

```python
def func(file_upload, file_mic):
    
    warn_output = ""
    if (file_mic is not None) and (file_upload is not None):
       warn_output = "WARNING: You've uploaded an audio file and used the microphone. The recorded file from the microphone will be used and the uploaded audio will be discarded.\n"
       file_name = file_mic
    elif (file_mic is None) and (file_upload is None):
       return "ERROR: You have to either use the microphone or upload an audio file"
    elif file_mic is not None:
       file_name = file_mic
    else:
       file_name = file_upload
```

<img width="975" alt="image" src="https://user-images.githubusercontent.com/52277510/174647622-502a4491-f236-45de-9ebc-04e014e94cba.png">

**Comment by pngwn** [comment]
This looks like a bug to me at a glance but could be missing something. @abidlabs @aliabid94 @FarukOzderim may know more.

**Comment by abidlabs** [comment]
I'm looking into this issue -- @sciai-ai what is the behavior that you expect in this case?

**Comment by abidlabs** [comment]
I believe this is fixed in #1722. Please let me know if that is not the case @sciai-ai.
